code/modules/mob/living/carbon/human/life.dm

```diff
@@ -79,6 +80,12 @@
 
 		handle_medical_side_effects()
 
+		if (life_tick % 4 == 1 && (get_game_time() >= dodge_time + 5 SECONDS))
+			if (confidence == FALSE)
+				to_chat(src, SPAN_NOTICE("You feel confident again."))
```

Comment:
That probably will be spammed in the chat a lot.

Reply:
only happens once, five seconds after the most recent attack; in other words you need to take a break from combat for at least five seconds to get this message.